proposals/IpfsGatewayHttpApi.md

```diff
@@ -0,0 +1,134 @@
+# IPFS Gateway HTTP API
+Authors: @anorth
+Initial PR: TBD
```

Comment:
Is this a self link to the PR that the file is created in, or initial development work for the proposal? If it's the former, it should be reasonably easy to look at history / git blame to find that initial conversation.

Reply:
This PR. I've inserted the link now the PR exists.